fontc_crater/src/main.rs

```diff
@@ -135,6 +136,22 @@ fn get_git_rev(repo_path: Option<&Path>) -> Option<String> {
     )
 }
 
+fn pip_freeze_sha() -> String {
+    let pipfreeze = Command::new("pip")
+        .arg("freeze")
+        .stdout(Stdio::piped())
+        .spawn()
+        .unwrap();
+    let sha1sum = Command::new("shasum")
+        .stdin(Stdio::from(pipfreeze.stdout.unwrap()))
+        .output()
+        .expect("shasum should be preinstalled everywhere");
+    assert!(sha1sum.status.success());
+    std::str::from_utf8(sha1sum.stdout.trim_ascii())
+        .expect("shasum output always ascii")
+        .to_owned()
+}
```

Comment:
You don't *really* need a crytographic hash or to spawn although I suppose it's harmless enough

Reply:
this was just the quickest way I could think of to do this in a consistent and portable way. 🤷